web_development_101/the_basics/introduction_to_web_development.md

```diff
@@ -1,50 +1,49 @@
-The work could involve the front end, the back end, or full stack.
+Web developers often work for clients who are trying to get their product or service onto the web.
+The work is typically very project focused and involves collaborating with a team that helps to coordinate the client's needs into the end product. The client could be a tech company, an organization, or a government. The work could involve front-end, back-end, or full-stack web development.
 
-This could be your profession if you like solving logical problems, building useful things, and experimenting with new technologies.
-Web developers are in high demand, generally have a good work/life balance, and command comfortable salaries. Google your specific location for more details.
+Web development could be a good profession for you if you like solving logical problems, building useful things, and experimenting with new technologies.
+Web developers are in high demand, generally have a good work/life balance, and command comfortable salaries. Google your specific location to get a better sense of your local web development job opportunities.
 
-For more details, Wikipedia lays out the breadth of the web design profession in their [entry on web design](https://en.wikipedia.org/wiki/Web_design).
+For more details, Wikipedia describes the breadth of the web development profession in their [entry on web design](https://en.wikipedia.org/wiki/Web_design).
 
 ### Types of Web Developers
 
-Earlier, we mentioned that the work could be in the front end, the back end, or full stack. What exactly are these?
+Earlier, we mentioned that web development work could be in the front end, the back end, or the full stack. What exactly do these terms mean?
 
-The "Front End" is the stuff you see on the website in your browser, including the presentation of content and the user interface elements like the navigation bar.
-Front End Developers make use of HTML, CSS, Javascript, and their relevant frameworks to ensure that content is presented effectively and that users have an excellent experience.
+The **front end** is the stuff you see on the website in your browser, including the presentation of content and user interface elements like the navigation bar.
+Front-end developers use HTML, CSS, JavaScript, and their relevant frameworks to ensure that content is presented effectively and that users have an excellent experience.
 
-The "Back End" refers to the guts of the application, which live on the server. They manipulate data appropriately to make sure the Front End has what it needs. This can become very complicated as things scale.
-Back End Developers make use of programming languages like Java, Python, and Ruby (among many others) to work with data.
+The **back end** refers to the guts of the application, which live on the server. The back end manipulates data to ensure that the front end has what it needs, which can become very complicated as things scale.
```

Comment:
```suggestion
The **back end** refers to the guts of the application, which live on the server. The back end stores and serves program data to ensure that the front end has what it needs. This can become very complicated when a website has millions of users.
```